readme.md

```diff
@@ -85,23 +91,28 @@ msbuild openrct2.proj /t:g2
 msbuild openrct2.proj /t:PublishPortable
 ```
 
-### Mac:
-We support native builds for macOS (limited to i386 only for now).
-Make sure that you have [Homebrew](http://brew.sh/) installed and than run the following commands to install all the needed libraries and build OpenRCT2.
-```
-# Install libraries
-./install.sh
+### macOS:
+#### Xcode:
+The recommended way of building OpenRCT2 for macOS is with Xcode. The Xcode build will create a self-contained application bundles which include all the necessary game files and dependencies. Open the project file OpenRCT2.xcodeproj in Xcode and build from there. Building this way will handle the dependencies for you automatically. You can also invoke an Xcode build from the command line using `xcodebuild`.
 
-# Build OpenRCT2
-./build.sh
+#### CMake:
+A command line version of OpenRCT2 can be built using CMake. This type of build requires you to provide the dependencies yourself. The supported method of doing this is with [Homebrew](http://brew.sh). Once you have Homebrew installed, you can download all the required libraries with this command:
```

Comment:
To be fair, I accidentally pushed this change, and I was going to review/discuss it further 😜 

While it is true that the Xcode build *can* be run from the command line, it's really not easy or intuitive. Plus, the CMake build is much more "command line" due to how it looks for the data files compared to when it's in the app bundle. I still think "command line" is the best way to describe it without adding a paragraph spelling out what an app bundle is and all the workarounds added to get the Xcode build to work without requiring the user to install dependencies system-wide.

Also, `make package` doesn't make an app bundle. I've tried it previously.